doc-content/jbpm-docs/src/main/asciidoc/Overview/processengine-overview-con.adoc

```diff
@@ -0,0 +1,45 @@
+[id='processengine-overview-con']
+= Overview of the {PROCESS_ENGINE}
+
+The {PROCESS_ENGINE} implements the Business Process Management (BPM) paradigm. BPM is a business methodology that enables modeling, measuring, and optimizing processes within an enterprise. 
+
+In BPM, a repeatable business process is represented as a workflow diagram. The Business Process Management Notation (BPMN) specification defines the available elements of this diagram. The {PROCESS_ENGINE} implements a large subset of the BPMN 2.0 specification.
+
+With {PROCESS_ENGINE}, business analysts can develop the diagram itself. Developers can implement the business logic and, when necessary, user interface of every element of the flow in code, making an executable business process. Users can execute the business process and interact with it as necessary. Then analysts can generate metrics that reflect the efficiency of the process.
+
+The workflow diagram consists of a number of nodes. The BPMN specification defined many kinds of nodes. The principal types of node are:
+
+* _Event_: nodes representing something happening outside of the process. Typical events are the start and the end of a process. An event can throw messages to other processes and catch such messages. Circles on the diagram represent events.
```

Comment:
inside or outside of the process (events could be internal to the process as well)